### PR TITLE
Prison Break Scenario [RDY]

### PR DESCRIPTION
**Pull request by DifferentLevelDan**
New Scenario. You start as convict in the prison basement. You
must fabricate tools to facilitate escape, and take care of the security
bots guarding the exit before the zombie brutes come crashing in.

### CODE REVIEW

**Comment by pisskop** [comment]
how do you suggest one get past the bars, get the materials to craft the tools to kill the bots?

Hav you done it?

**Comment by Rod-Serling** [comment]
Bots can be avoided (or take electronics skill and "disassemble" computers to craft EMP grenade with a lucky soldering iron drop). Additionally the easy-to-access armory sometimes have the necessary equipment to neutralize them in low-light conditions.

You can either craft lockpicks, a quarterstaff, or a homewrecker to bash through the metal bar doors. 

Yes I have done it. Hence the addition of matches to the convict profession so they can have some light for crafting.

You generally spawn in the large common area in the basement of the prison, not in a cell. There's two stoves, three refrigerators, a bed, several lockers, plenty of wood tables and chairs, several lockers, and toilets, all behind only locked wood doors that can be easily bashed by hand. Plenty of food and starting drink in the kitchen area. It's doable, but not necessarily easy, and if a zombie brute spawns in the common area with you, you might have a hard time. Lab challenge has spawned me in rooms with turrets though, so I think it's fair.

File: data/json/scenarios/scenarios.json [hunk 1]
```diff
@@ -101,6 +101,16 @@
     "professions" : ["unemployed","bionic_patient","patient","broken_cyborg"],
     "flags" : ["CHALLENGE"]
   },{
+    "type" : "scenario",
+    "ident" : "prisonbreak",
+    "name" : "Prison Break",
+    "points" : -4,
+    "description" : "You were a convicted felon planning your escape before the event. Now the prisoners have turned into mindless horrors and the security bots are shooting on sight... you might need to expedite your escape plans.",
```
Comment (Night-Pryanik) [inline data/json/scenarios/scenarios.json]: Two spaces after period, please.

**Comment by pisskop** [comment]
would it be tacky to give him ar rockhammer?

**Comment by Rivet-the-Zombie** [comment]
>would it be tacky to give him ar rockhammer?

Look for it behind the poster of Rita Hayworth.

File: data/json/scenarios/scenarios.json [hunk 2]
```diff
@@ -101,6 +101,16 @@
     "professions" : ["unemployed","bionic_patient","patient","broken_cyborg"],
     "flags" : ["CHALLENGE"]
   },{
+    "type" : "scenario",
+    "ident" : "prisonbreak",
+    "name" : "Prison Break",
+    "points" : -4,
+    "description" : "You were a convicted felon planning your escape before the event.  Now the prisoners have turned into mindless horrors and the security bots are shooting on sight... you might need to expedite your escape plans.",
```
Comment (illi-kun) [inline data/json/scenarios/scenarios.json]: It might be good to replace set of three points ("...") by a proper ellipsis symbol ("…").

**Pull request merged**